src/transformers/tokenization_auto.py

```diff
@@ -101,6 +103,7 @@
         (ElectraConfig, (ElectraTokenizer, ElectraTokenizerFast)),
         (FunnelConfig, (FunnelTokenizer, FunnelTokenizerFast)),
         (LxmertConfig, (LxmertTokenizer, LxmertTokenizerFast)),
+        (LayoutLMConfig, (LayoutLMTokenizer, None)),
```

Comment:
I think we can add a fast version too since it just inherits from the bert tokenizer.